KAZARI_PLUGIN.md

```diff
@@ -0,0 +1,77 @@
+# Kazari
+
+Kazari is a JavaScript plugin developed on **Scala.Js** that allows to turn your project's existing documentation into interactive Scala worksheets. Basically it decorates your code snippets with extra functionality that allows users to run them in a [remote Scala evaluator](https://github.com/scala-exercises/evaluator) and edit it, by just including the script and some CSS files to your site.
+
+# Installation
+
+In order to use Kazari, you first need to include two scripts:
+
+* `sbt-microsites-js-opt.js`, containing the main code of the decorator plugin.
```

Comment:
I'll try to play with this idea a little bit to see if it works by creating a custom SBT command, it'd certaintly be nicer.